partcad/src/partcad/sentry.py

```
@@ -0,0 +1,135 @@
+
+        provider = TracerProvider()
+        provider.add_span_processor(SentrySpanProcessor())
+        trace.set_tracer_provider(provider)
+        set_global_textmap(SentryPropagator())
+
+
+tracer = trace.get_tracer("PartCAD")
+
+
+@asynccontextmanager
+async def start_as_current_span_async(name, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for public function `start_as_current_span_async`

(ANN201)